stubs/TYPO3/CMS/Core/Database/Query/QueryBuilder.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare(strict_types=1);
```

Comment:
According to code style, there should be an empty line before and after declare statements.

Reply:
It's pretty annoying that some of that stuff seems to not get applied, isn't it?
EOL, FQN in config files and now this